website/content/docs/k8s/connect/terminating-gateways.mdx

```diff
@@ -88,6 +88,54 @@ Registering the external services with Consul is a multi-step process:
 
 ### Register external services with Consul
 
+There are two ways to register an external service with Consul:
+1. If you have [`TransparentProxy`](/docs/k8s/helm#v-connectinject-transparentproxy) enabled, you can declare external endpoints with the [Destination](/docs/connect/config-entries/service-defaults#terminating-gateway-destination) field of `service-defaults`.
+1. You can add the service as a Node in the Consul catalog.
+
+#### Register an external service as a Destination
+
+Destinations are valid only for [`TransparentProxy`](/docs/k8s/helm#v-connectinject-transparentproxy) mode and allow clients to dial the external service directly.
+Also note these additional limitations when using Destinations to route traffic through a terminating gateway:
+
+| Layer 4 External Services (TCP) | Client dials hostname | Client dials IP |
+|---|---|---|
+| **with TLS** | ✅ Clients can dial successfully. `CAFiles` are not allowed because traffic is already end-to-end encrypted by the client. | ✅ Clients can dial successfully. CAFiles are not allowed because traffic is already end-to-end encrypted by the client. |
+| **no TLS** | ❌ Because the Sidecar is not protocol aware, it will not be able to identify traffic going to the external service. | ✅ Clients can dial successfully. |
```

Comment:
For IP with no TLS we should say that you can encrypt traffic at the tgtw using SNI and certs.